app/feedme/search/hybrid_search_supabase.py

```
@@ -0,0 +1,307 @@
+"""
+FeedMe v2.0 Hybrid Search Engine with Supabase Integration
+Combines vector similarity and full-text search with Supabase support
+"""
+
+import logging
+import time
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)